src/main/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionActor.scala

```diff
@@ -147,6 +147,7 @@ trait WorkflowSubmission extends FutureSupport with LazyLogging with MethodWiths
       user.userEmail.value,
       token,
       billingProject.cromwellAuthBucketUrl,
+      s"gs://${workspace.bucketName}/${submissionId}/workflow-logs",
```

Comment:
so this path could overlap with a path that cromwell generates if someone were to run a method called "workflow-logs", but as far as i know it won't actually conflict with any of their data